src/api/utils/auth.py

```diff
@@ -48,9 +49,10 @@ def decorated(*args, **kwargs):
                                  options={
                                      'verify_signature': True,
                                      'verify_exp': True
-                                 }
+                                 },
+                                 audience=current_app.config['API_IDENTIFIER'],
+                                 issuer=current_app.config['API_IDENTIFIER']
```

Comment:
Could you walk me through this as well?

Reply:
@Andretalik Tech team changed their token structure to include that but those changes have been reverted.